.github/workflows/spelling.yml

```diff
@@ -0,0 +1,30 @@
+name: Spell checking
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  pull_request_target:
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    name: Spell checker
+    runs-on: ubuntu-20.04
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+      with:
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: check-spelling
+      uses: ./
+      with:
+        experimental_apply_changes_via_bot: 1
```

Comment:
According to the [wiki page](experimental_apply_changes_via_bot)
> this feature is not ready for repositories with non members/other bots

I think cve-bin-tool falls into this category, we have codecov bot commenting on PRs.

Additionally there's no newline at end of file
![image](https://user-images.githubusercontent.com/12860284/130270911-80bf3594-1304-4063-a34f-498abab5efc8.png)